packages/in_app_purchase/test/fake_in_app_purchase_platform.dart

```diff
@@ -1,9 +1,10 @@
 import 'dart:async';
 import 'package:flutter/services.dart';
 
-class FakePlatformViewsController {
+class FakeInAppPurchasePlatform {
```

Comment:
As discussed, we think that this is ore of a stub than a fake, lets name appropriately.